<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="UTF-8">
    <title>Matchmating for Computational and Experimental Biologists Session - GLBIO 2017 by MiBatGLBIO</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
  </head>
  <body>
    <section class="page-header">
      <h1 class="project-name">Matchmating for Computational and Experimental Biologists Session - GLBIO 2017</h1>
      <h2 class="project-tagline">@GLBIO2017 @BBSD2017</h2>
      <a href="https://github.com/MiBatGLBIO/GLBIO_2017" class="btn">View on GitHub</a>
      <a href="https://github.com/MiBatGLBIO/GLBIO_2017/zipball/master" class="btn">Download .zip</a>
      <a href="https://github.com/MiBatGLBIO/GLBIO_2017/tarball/master" class="btn">Download .tar.gz</a>
    </section>

    <section class="main-content">
      <h2>
<a id="glbio-2017-special-session" class="anchor" href="#glbio-2017-special-session" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>GLBIO 2017 Special Session</h2>

<h2>
<a id="httpswwwiscborgglbio2017" class="anchor" href="#httpswwwiscborgglbio2017" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>(<a href="https://www.iscb.org/glbio2017">https://www.iscb.org/glbio2017</a>)</h2>

<h2>
<a id="matchmaking-for-computational-and-experimental-biologists-session" class="anchor" href="#matchmaking-for-computational-and-experimental-biologists-session" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Matchmaking for Computational and Experimental Biologists Session</h2>

<p>The Matchmaking@GLBIO session (Matchmaking for Computational and Experimental Biologists Session) will have 3 short talks about specific tools and resources on RNA-seq, chromatin accessibility, and omic integration.  Next, tool developers/analysis experts will interact with bench scientists/novice researchers to discuss common interests and specific projects, with the goal of establishing future conversations and/or collaborations. </p>

<h2>
<a id="how-to-participate" class="anchor" href="#how-to-participate" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>How to participate</h2>

<h3>
<a id="instructions-of-tool-developers-bioinformaticists" class="anchor" href="#instructions-of-tool-developers-bioinformaticists" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Instructions of tool developers (Bioinformaticists)</h3>

<p>If you are interested in participating and gaining feedback and visibility on your tool, please submit and abstract though the <a href="https://docs.google.com/forms/d/e/1FAIpQLScb6N8Hd2rJ-8sJQS9eGDQZYCGeOohCHyhGKQiqZllf7fwEbQ/viewform">following link</a>.  Tools should be applicable to RNA-seq data, chromatin accessibility data, or omics integration.</p>

<h3>
<a id="instructions-for-tool-users-biologists-clinicians-etc" class="anchor" href="#instructions-for-tool-users-biologists-clinicians-etc" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Instructions for tool users (Biologists, Clinicians, etc.)</h3>

<p>If you have some data that you need help analyzing, or if you're trying to learn about the latest available tools in the RNA-seq data analysis, chromatin accessibility data analysis, or omics integration, fill out our form <a href="https://docs.google.com/forms/d/e/1FAIpQLScb6N8Hd2rJ-8sJQS9eGDQZYCGeOohCHyhGKQiqZllf7fwEbQ/viewform">here</a>.</p>

<h2>
<a id="schedule" class="anchor" href="#schedule" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Schedule</h2>

<p><b><strong>3 - 15 min topic talks</strong></b></p>

<ul>
<li>Ben Busby, NIH -- ​Simple Computational Tools for RNA-seq Isoform Analysis​</li>
<li>Ewy Mathe, OSU -- ​Chromatin Accessibility Analysis for Biological/Clinical Applications</li>
<li>TBD -- ​Omic Data Integration with Popular Bioinformatic Tools</li>
</ul>

<p><b> <strong>Speed Dating Session</strong></b></p>

<p>Clinicans and biologists meet for 4 minutes with tool developers to understand their expertise and determine which developers they should be following up with. This will foster communication between the groups, and will allow tool users and developers to identify potential collaborators. This approach has been recently applied at the O’Reilly 2016 AI conference and was very well received.</p>

<p><b> <strong>Small Group Meetings</strong></b></p>

<p>Participants will meet in small groups and pair up with the expert(s) in one of the 3 themes (RNA-seq, chromatin accessibility, omics integration). In these small group sessions, group leaders (the experts) will highlight workflows that are accessible to novice users, will discuss best practices, and will answer questions that the participants may have.</p>

<h2>
<a id="organizers" class="anchor" href="#organizers" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Organizers</h2>

<p><b>Ewy Mathé, Ohio State University, <a href="mailto:ewy.mathe@osumc.edu">ewy.mathe@osumc.edu</a> </b><br>
<a href="u.osu.edu/mathelab">Lab website</a><br>
<a href="github.com/mathelab">Github website</a><br>
Dr. Mathé’s primary research interest are to leverage epigenomics, genomics, nucleotide variants and metabolic patterns to 1) understand how the genetic and epigenetic landscape affects disease phenotypes, particularly cancer; 2) define cell-type and disease-type specific molecular characteristics to uncover novel biomarkers and guide the search of novel therapeutic targets.</p>

<p><b>Helen Piontkivska, Kent University, <a href="mailto:opiontki@kent.edu">opiontki@kent.edu</a> </b><br>
<a href="https://scholar.google.com/citations?user=fb2iQfwAAAAJ&amp;hl=en">Google Scholars</a> <br>
<a href="https://www.kent.edu/biology/profile/helen-piontkivska">Lab Website</a><br>
Dr. Piontkivska’s expertise is in molecular evolutionary genetics, genomics and bioinformatics research, including data mining and machine learning. Her lab is working on understanding the mechanisms of adaptive evolution of genomes, including the evolution of adaptive immune mechanisms and evolutionary consequences of host-pathogen interactions. She also uses personalized genomics data, including exome sequencing, and large-scale expression profiling to identify phenotype-specific biomarkers (e.g., in cancer).</p>

<p><b>Ben Busby, NCBI, NLM, NIH,  <a href="mailto:ben.busby@nih.gov">ben.busby@nih.gov</a>) </b><br>
<a href="github.com/NCBI-hackathons">Github site</a><br>
<a href="https://github.com/DCGenomics" class="user-mention">@DCGenomics</a><br>
Dr. Busby’s expertise is in genomics and bioinformatics research, tool generation, testing and education.</p>

      <footer class="site-footer">
        <span class="site-footer-owner"><a href="https://github.com/MiBatGLBIO/GLBIO_2017">Matchmating for Computational and Experimental Biologists Session - GLBIO 2017</a> is maintained by <a href="https://github.com/MiBatGLBIO">MiBatGLBIO</a>.</span>

        <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
      </footer>

    </section>

  
  </body>
</html>
