## GLBIO 2017 Special Session
## (https://www.iscb.org/glbio2017)

## Matchmaking for Computational and Experimental Biologists Session

The Matchmaking@GLBIO session (Matchmaking for Computational and Experimental Biologists Session) will have 3 short talks about specific tools and resources on RNA-seq, chromatin accessibility, and omic integration.  Next, tool developers/analysis experts will interact with bench scientists/novice researchers to discuss common interests and specific projects, with the goal of establishing future conversations and/or collaborations. 

## How to participate
### Instructions of tool developers (Bioinformaticists)
If you are interested in participating and gaining feedback and visibility on your tool, please submit and abstract though the [following link](https://docs.google.com/forms/d/e/1FAIpQLScb6N8Hd2rJ-8sJQS9eGDQZYCGeOohCHyhGKQiqZllf7fwEbQ/viewform).  Tools should be applicable to RNA-seq data, chromatin accessibility data, or omics integration.

### Instructions for tool users (Biologists, Clinicians, etc.)
If you have some data that you need help analyzing, or if you're trying to learn about the latest available tools in the RNA-seq data analysis, chromatin accessibility data analysis, or omics integration, fill out our form [here](https://docs.google.com/forms/d/e/1FAIpQLScb6N8Hd2rJ-8sJQS9eGDQZYCGeOohCHyhGKQiqZllf7fwEbQ/viewform).

## Schedule
<b>__3 - 15 min topic talks__
* Ben Busby, NIH -- ​Simple Computational Tools for RNA-seq Isoform Analysis​
* Ewy Mathe, OSU -- ​Chromatin Accessibility Analysis for Biological/Clinical Applications
* James Chen, OSU -- ​Integrating Genomics and Clinical Outcomes Data to Optimize Patient Matching

<b> __Speed Dating Session__

Clinicans and biologists meet for 4 minutes with tool developers to understand their expertise and determine which developers they should be following up with. This will foster communication between the groups, and will allow tool users and developers to identify potential collaborators. This approach has been recently applied at the O’Reilly 2016 AI conference and was very well received.

<b> __Small Group Meetings__

Participants will meet in small groups and pair up with the expert(s) in one of the 3 themes (RNA-seq, chromatin accessibility, omics integration). In these small group sessions, group leaders (the experts) will highlight workflows that are accessible to novice users, will discuss best practices, and will answer questions that the participants may have.

## Organizers
<b>Ewy Mathé, Ohio State University, ewy.mathe@osumc.edu </b><br>
[Lab website](u.osu.edu/mathelab)
<br>
[Github website](github.com/mathelab)
<br>
Dr. Mathé’s primary research interest are to leverage epigenomics, genomics, nucleotide variants and metabolic patterns to 1) understand how the genetic and epigenetic landscape affects disease phenotypes, particularly cancer; 2) define cell-type and disease-type specific molecular characteristics to uncover novel biomarkers and guide the search of novel therapeutic targets.

<b>Helen Piontkivska, Kent University, opiontki@kent.edu </b><br>
[Google Scholars](https://scholar.google.com/citations?user=fb2iQfwAAAAJ&hl=en) <br>
Dr. Piontkivska’s expertise is in molecular evolutionary genetics, genomics and bioinformatics research, including data mining and machine learning. Her lab is working on understanding the mechanisms of adaptive evolution of genomes, including the evolution of adaptive immune mechanisms and evolutionary consequences of host-pathogen interactions. She also uses personalized genomics data, including exome sequencing, and large-scale expression profiling to identify phenotype-specific biomarkers (e.g., in cancer).

<b>Ben Busby, NCBI, NLM, NIH,  ben.busby@nih.gov </b><br>
[Github site](github.com/NCBI-hackathons)<br>
@DCGenomics<br>
Dr. Busby’s expertise is in genomics and bioinformatics research, tool generation, testing and education.
